vpr/src/noc/read_xml_noc_traffic_flows_file.h

```diff
@@ -93,8 +145,10 @@ void process_single_flow(pugi::xml_node single_flow_tag, const pugiutil::loc_dat
 void verify_traffic_flow_router_modules(std::string source_router_name, std::string sink_router_name, pugi::xml_node single_flow_tag, const pugiutil::loc_data& loc_data);
 
 /**
- * @brief Ensures that the a given traffic flows data transmission size and
- *        latency constraints are not negative values.
+ * @brief Ensures that the a given traffic flows bandwidth and
```

Comment:
flows -> flow's
Could shorten: ensures the traffic flow's bandwidth, latency constraint and priority are all non-negative.

Reply:
Fixed